src/PrestaShopBundle/Install/Install.php

```diff
@@ -449,6 +450,7 @@ public function installDefaultData($shop_name, $iso_country = false, $all_langua
                         }
                     }
                 }
+                $iso_codes_to_install[] = self::FALLBACK_LANGUAGE;
```

Comment:
As I said, always installing English is a bad idea. All our merchants do not need this language to be installed and may remove it anyway.

Reply:
ok I will drop this modification, and get back the required changes in the other PR (which removes useless xml files)